DEVNOTES.md

```diff
@@ -17,8 +17,20 @@ cd consent
 mvn clean test package 
 ```
 
-Tests spin up embedded postgres and http servers that run against localhost.
-Ensure that your test environment supports that.
+Tests require docker or another OCI runtime to spin up postgres and http servers that run against
+localhost. Ensure that your test environment supports that.  Command line docker, podman, and 
+Docker Desktop have all worked to run these tests.  The project officially supports docker, but the
+example below gives you an idea of how to use podman.  
+
+For example:
+```
+% brew install podman
+% podman machine
+% podman machine init --cpus 8 -m 4096 --now
+```
+Depending on your system configuration, you may need to set the DOCKER_HOST environment variable.
+Check the output of the `podman machine init` command.
+
```

Comment:
Suggestion: Move the added text to the Docker section. Also, I'm not sure if we should specifically call out how to install `podman`, since different developers have different types of systems and this only applies to macOS. On the other hand, it may be worth emphasizing that running with `podman` provided a performance advantage for these tests.

Reply:
I'm inclined to leave the intro to the testing section, because to run the tests, you do need an OCI runtime (e.g., docker, podman, etc) running - but I like your other suggestions.  Let me know if my update works for you.